scss/forms/_form-range.scss

```
@@ -34,7 +34,7 @@
     box-sizing: content-box; // Boosted mod
     width: $form-range-thumb-width;
     height: $form-range-thumb-height;
-    margin-top: ($form-range-track-height - $form-range-thumb-height) * .5; // Webkit specific
+    margin-top: subtract(($form-range-track-height - $form-range-thumb-height) * .5, $form-range-thumb-focus-box-shadow-width); // Webkit specific // Boosted mod
```

I feel weird to use a box-shadow-width here. I feel like `$border-width` would be more appropriate here + here what was done in #1282.
```suggestion
    margin-top: ($form-range-track-height - ($form-range-thumb-height + $border-width * 2)) * .5; // Webkit specific // Boosted mod
```
Here, `$form-range-thumb-height + $border-width * 2` is the calculed height of the thumb with webkit.